docs/dev/service_levels.md

```diff
@@ -134,3 +135,20 @@ Otherwise, e.g. if a role has multiple workload types declared,
 the conflicts are resolved as follows:
  - `X` vs `unspecified` -> `X`
  - `batch` vs `interactive` -> `batch` - under the assumption that `batch` is safer, because it would not trigger load shedding as eagerly as `interactive`
+
+### Effective service level
+
+User may have assigned multiple service levels by granting one role to another.
+
+As mentioned previously, any user may have assgined multiple service levels by granting one roles to another. To facilitate insight into which values come from which service level, there is `LIST EFFECTIVE SERVICE LEVEL OF <role_name>` command.
```

Comment:
typo "assgined" and "roles".
Also, no need to repeat the exact same sentence ("user may have assigned multiple service levels by granting oen role to another")., so please copy-edit this text.

Reply:
changed that